management/main_manage.py

```diff
@@ -109,7 +122,9 @@ def suck(self,old_mail):
         for msg in old_mail.storytime:
             self.story(msg)
         for parcel in old_mail.channel:
-            self.respond(msg)   
+            self.channel.append(parcel)
```

Comment:
is DiscordChannel.append() a method? Is this rewrite?

Reply:
No, this may be a little confusing, sorry. As you could see up above, the Mailbox() has five tables, among which channel and player. The list channel is, basically, a list of messages that need to be sent to certain channels.
So it's not DiscordChannel.append(), it's ListofmessagestoDiscordChannels.append().

Reply:
We could change its name. I don't think we're using it outside of the class' definition.